services/src/main/java/org/keycloak/protocol/oidc/grants/JWTAuthorizationGrantType.java

```diff
@@ -64,7 +63,6 @@ public Response process(Context context) {
             authorizationGrantContext.validateClient();
 
             //mandatory claims
-            authorizationGrantContext.validateTokenAudience(List.of(expectedAudience), false);
```

Comment:
You have to remove the definition of this `expectedAudience` variable above, right?

Reply:
Yes, you are right. Done!